### DISCUSSION TITLE
Question 21: Sites country ISO2 field not automatic? [WCG]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#997] [Q&A]
Description: In the Sites->Class:Political tab, the ISO2 field (“AR” for Argentina etc) is editable instead of inheriting from the parent. Are we supposed to be manually filling this code in for every site we make? I have not been

**Comment by rondlg** [comment]
Response:  This should automatically fill in if the parent has its ISO value filled in. If not let us know and we’ll submit a hornbill ticket.

**Reply by wyattgaswick** [reply]
So...I couldn't recreate this issue (creating new sites does inherit the parent's ISO2 field). _BUT_ there are a lot with missing ISO2 fields, and using any of these as a parent perpetuates the error to the new record. The most recent site IRN I can find with this is from 2 days ago (irn 740012). Its parent (699352) has no ISO2 code, but the parent of _that_ record (17296) is fine.

If it's possible to fill these in, that would be great and help prevent the issue from propagating. I count 130,847 affected records in Botany and FMNH Master, not sure how many in other areas.
